### DISCUSSION TITLE
Crazy question

### DISCUSSION THREAD

**Discussion by jamo-lcr** [geerlingguy/raspberry-pi-pcie-devices#145] [General]
Hello i have an crazy question for the rasbarry Pi ist it possible to remove the cpu Like in this Video https://youtu.be/9yrncWd0Iag  and Set a faster better cpu with the Same pinouts in the raspbarry  Pi I dont Know the pinouts  for Boardcom 2711 but ist there a possebiltity to build a Snapdragon 8... In ?Maby there are language mistages because i am not an Englisch speaker.

**Comment by 6by9** [comment]
No. SoCs don't have a standardised ball-out, same as AMD and Intel have incompatible CPU sockets.
